testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/drone/KeycloakDronePostSetup.java

```diff
@@ -102,6 +109,30 @@ private void configureFirefoxDriver(WebDriver driver) {
         }
     }
 
+    private void configureHtmlUnitDriver(WebDriver driver) {
+        if (driver instanceof HtmlUnitDriver) {
+            HtmlUnitDriver htmlUnitDriver = (HtmlUnitDriver) driver;
+
+            final var keystore = System.getProperty(HTML_UNIT_SSL_KEYSTORE_PROP);
+            final var keystorePassword = System.getProperty(HTML_UNIT_SSL_KEYSTORE_PASSWORD_PROP);
+            final var keystoreType = System.getProperty(HTML_UNIT_SSL_KEYSTORE_TYPE_PROP);
+
+            log.info("Check HtmlUnit driver TLS settings");
+
+            if (keystore != null && keystorePassword != null && keystoreType != null) {
```

Comment:
This could also be an early return to reduce nesting.
```java
if (keystore == null && keystorePassword == null && keystoreType == null) {
    return;
}
```